src/sentry/data_export/processors/discover.py

```diff
@@ -101,4 +111,11 @@ def handle_fields(self, result_list):
             for result in new_result_list:
                 if "issue.id" in result:
                     result["issue"] = issues.get(result["issue.id"], "unknown")
+
+        # Map equations back to their unaliased forms
+        if self.equation_aliases:
+            for result in new_result_list:
+                for equation_alias, equation in self.equation_aliases.items():
+                    result[equation] = result.get(equation_alias)
```

Comment:
The equations can contain commas right? Wouldnt that be problematic in the resulting csv?

Reply:
hmmm can't find anything on the csv.writer docs, but tested this and can confirm this is handled correctly by the csvwriter, so a header containing a comma will be wrapped in quotes so that it is treated as a single column